source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,842 @@
-with the default values from the `ister` package with the following command:
+.. rst-class:: content-collapse
 
-.. code-block:: bash
+Configure Docker proxy info
+===========================
 
-   sudo cp /usr/share/defaults/ister/ister.json release-image-config.json
+If needed, use these steps to configure the Docker proxy information.
 
-For reference, you can inspect the `Clear Linux ister configuration file`_
-used for releases.
+Configure the Docker daemon proxies:
 
-Edit the configuration file to include all bundles you want *preinstalled* in
-the image. Users can install the bundles in the mix that are not included in
-the configuration file with the following command:
+#. Create the Docker daemon proxy config directory:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   sudo swupd bundle-add bundle1
+      sudo mkdir -p /etc/systemd/system/docker.service.d
+  
+   Create :file:`/etc/systemd/system/docker.service.d/http-proxy.conf` and
+   add the following using your own proxy values:
 
-Keeping the list of bundles in the configuration file small allows for a
-smaller image size. For the minimal base image, the list is:
+   .. code-block:: console
 
-.. code-block:: console
+      [Service]
+      Environment="HTTP_PROXY=<HTTP proxy URL>:<port number>"
+      Environment="HTTPS_PROXY=<HTTPS proxy URL>:<port number>"
 
-   "Bundles": ["os-core", "os-core-update", "kernel-native"]
+#. Reload the Docker daemon:
 
-Next, set the `Version` field to the mix version that you want the content
-mixer to use to build the image. `ister` allows you to build an image from
-any mix version that you have built, not just from the current version. In
-our example so far, `Version` is set to 10.
+   .. code-block:: bash
 
-With the `ister` tool configured, build the image with the following command:
+      sudo systemctl daemon-reload
 
-.. code-block:: bash
+Configure the Docker container proxies, in order to pass proxy
+settings to containers:
 
-   sudo mixer build image
+#. Create a directory for your container config:
 
-Mixer automatically looks for the :file:`release-image-config.json` file, but
-you can freely choose the filename. To use a different name, simply pass the
-*--template* flag when creating your image, for example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      mkdir ~/.docker
 
-   sudo mixer build image --template path/to/file.config
+#. Create the config file :file:`~/.docker/config.json` and add the following
+   entries, using your own proxy values:
 
-By default, `ister` uses the format version of the build machine it runs on.
-Therefore, if the format you are building differs from the format of the |CL|
-OS you are building on, you must use the *--format <FORMAT_NUMBER>*
-flag. Find the current format version of your OS with the following command:
+   .. code-block:: console
 
-.. code-block:: bash
+      {
+        "proxies":
+        {
+          "default":
+          {
+            "httpProxy": ":",
```

Comment:
This should be:
"httpProxy": "<HTTP PROXY URL>:<port number>"
"httpsProxy": "<HTTPS PROXY URL>:<port number>"

Reply:
I think I may finally have this right.